src/App.css

```diff
@@ -1,38 +1,253 @@
+/* COLOUR PALETTE */
+/* #FF9F1C */
+/* #FFBF69 */
+/* #FFFFFF */
+/* #CBF3F0 */
+/* #2EC4B6 */
+
+/* FONTS */
+@import url('https://fonts.googleapis.com/css2?family=Alegreya+Sans+SC:wght@400;700&family=Halant&family=Iceberg&display=swap');
+/* font-family: 'Alegreya Sans SC',
+sans-serif;
+
+font-family: 'Halant',
+serif;
+
+font-family: 'Iceberg',
+cursive; */
+
 .App {
-  text-align: center;
+  background-color: #CBF3F0;
+  min-height: 110vh;
```

Comment:
Just curious why this has a height of 110vh? Would look just slightly cleaner with 100vh. Understandable though if it's a spacing issue.